woocommerce/woocommerce
# remove held time check from coupon usage limit

### All Submissions:






### Changes proposed in this Pull Request:



Remove the hold stock time check from coupon limitation check. It will disallow the coupon to be used more than the specified number of times per guest user.

Closes #26002

### How to test the changes in this Pull Request:

 1. Set Hold Stock to blank.
 2. Create a new coupon that has a Usage limit per user of 1.
 3. Add product to cart as guest, apply coupon, and check out.
 4. Repeat step 3 using the same email, which will allow you to check out again, but shouldn't.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Remove the hold stock time check from coupon limitation check. It will disallow the coupon to be used more than the specified number of times per guest user.
